scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontendState.java

```diff
@@ -1230,4 +1231,22 @@ public UserData getCurrentUserData() throws NotConnectedException {
         renewUserSession(id, ident);
         return PASchedulerProperties.getPropertiesAsHashMap();
     }
+
+    synchronized TaskStatesPage getTaskPaginated(JobId jobId, int offset, int limit)
+            throws UnknownJobException, NotConnectedException, PermissionException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)